src/foundation.cs

```diff
@@ -3173,6 +3173,12 @@ public interface NSExpression : NSSecureCoding, NSCopying {
 		[Static, Export ("expressionForFunction:arguments:")]
 		NSExpression FromFunction (string name, NSExpression[] parameters);
 
+		[Static, Export ("expressionWithFormat:")]
+		NSExpression FromFormat (string expressionFormat);
+		
+		[Static, Internal, Export ("expressionWithFormat:arguments:", IsVariadic = true)]
+		NSExpression FromFormat (string expressionFormat, IntPtr argList);
```

Comment:
I'm seen the code. I'm moving to the [] method and just expose the params one to follow Apples api.